<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmMain</class>
 <widget class="QWidget" name="frmMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>409</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>报销工具</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../delphi-workspace/mailsender/favicon.ico</normaloff>../../delphi-workspace/mailsender/favicon.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labServer">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>服务器:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cboxServer">
       <property name="minimumSize">
        <size>
         <width>290</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>290</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>smtp.qiye.163.com</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labPort">
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>55</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>端口:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cboxPort">
       <item>
        <property name="text">
         <string>25</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ckSSL">
       <property name="text">
        <string>SSL</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="labSender">
       <property name="text">
        <string>用户名:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtSender">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labSenderPwd">
       <property name="text">
        <string>密码:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtSenderPwd">
       <property name="text">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>60</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>报  销:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="month">
       <item>
        <property name="text">
         <string>1月</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2月</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3月</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4月</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5月</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6月</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7月</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8月</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9月</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10月</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>11月</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>12月</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="labAtta">
       <property name="text">
        <string>附  件:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtAtta"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnSelect">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>浏览</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="btnSend">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
